app/assets/javascripts/initializers/initializeCommentsPage.js.erb

```diff
@@ -557,8 +557,12 @@ function listenForDetailsToggle() {
 }
 
 function updateCommentsCount() {
-  var commentsCountDiv = document.getElementsByClassName("js-comments-count")[0];
-  var commentsCountData = parseInt(commentsCountDiv.dataset.commentsCount) + 1;
+  const commentsCountDiv = document.getElementsByClassName("js-comments-count")[0];
+
+  // if there's nowhere to put the count return early.
+  if(!commentsCountDiv) return;
+
+  const commentsCountData = parseInt(commentsCountDiv.dataset.commentsCount) + 1;
```

Comment:
Refactor (blocking): Add a [radix](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/parseInt#parameters) to `parseInt` for base 10, e.g. `parseInt(commentsCountDiv.dataset.commentsCount, 10)` or use `Number(commentsCountDiv.dataset.commentsCount)`

![image](https://user-images.githubusercontent.com/833231/117214390-bbc23c80-adca-11eb-9bc3-a7d03339178b.png)